lib/vintage_net_lte.ex

```diff
@@ -67,9 +75,9 @@ defmodule VintageNetLTE do
     ]
   end
 
-  defp chatscript_path(opts) do
+  defp chatscript_path(ifname, opts) do
     opts
     |> Keyword.fetch!(:tmpdir)
-    |> Path.join(Twilio.name())
+    |> Path.join(ifname)
```

Comment:
How about making the filename be `chatscript.#{ifname}`. That will be consistent with other config files.